### PR TITLE
Port linworks/dolice to Vial

### PR DESCRIPTION
**Pull request by TChanhTinh**

### CODE REVIEW

File: keyboards/linworks/dolice/config.h [hunk 1]
```diff
@@ -41,4 +41,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define LED_NUM_LOCK_PIN B6
 #define LED_CAPS_LOCK_PIN C7
 #define LED_SCROLL_LOCK_PIN C6
-#define LED_PIN_ON_STATE 0
\ No newline at end of file
+#define LED_PIN_ON_STATE 0
+
+// VIAL
+#define VIAL_KEYBOARD_UID {0xB6, 0xA6, 0x3B, 0x20, 0x22, 0xCE, 0xE9, 0x6D}
```
Comment (xyzz) [inline keyboards/linworks/dolice/config.h]: This should go into the vial keymap as per the porting guide

**Review by xyzz — CHANGES_REQUESTED** [review]

File: keyboards/linworks/dolice/config.h (hunk 1 above)
Comment (TChanhTinh) [reply keyboards/linworks/dolice/config.h]: Sorry for the delay, I've moved the the vial setting to config.h inside vial folder

File: keyboards/linworks/dolice/config.h [hunk 2]
```diff
@@ -41,4 +41,4 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define LED_NUM_LOCK_PIN B6
 #define LED_CAPS_LOCK_PIN C7
 #define LED_SCROLL_LOCK_PIN C6
-#define LED_PIN_ON_STATE 0
```
Comment (xyzz) [inline keyboards/linworks/dolice/config.h:44]: there's still a diff in this file, please undo the change

**Review by xyzz — CHANGES_REQUESTED** [review]
